addons/backup-and-maintenance.sh

```diff
@@ -37,10 +37,12 @@ NODE2_HOSTNAME=''
 NODE1_IP=''
 NODE2_IP=''
 
-# Only for MariaDB remote clusters
-MARIADB_REMOTE_CLUSTER=0
-# should be equal to return of hostname command
-MARIADB_BACKUP_SERVER_HOSTNAME=''
+# to detect MariaDB remote DB
+if [ "$DB_HOST" != "localhost" ] && [ "$DB_HOST" != "100.64.0.1" ]; then
```

Comment:
if 127.0.0.1 ?

Reply:
Is `127.0.0.1` a value still valid in 12.0.0 ?

I suppose it's when you are using `haproxy-db` in place of `proxysql`

Reply:
it´s just in case we forgot to change the value in the conf. Just to prevent no backup just because of that.

Reply:
127.0.0.1 is not a valid value anymore in pf.conf whether its haproxy-db or proxysql (there is a FATAL checkup in a cluster for that)

The setup will be hosed if that value is left there so I think a non-working backup will be the least of your trouble if 127.0.0.1 is left there